### PR TITLE
Add block_fade event

### PR DESCRIPTION
**Pull request by EntryPointKR**
Add block_fade event

### CODE REVIEW

**Comment by PseudoKnight** [comment]
Is there any case where data won't be 0?

File: src/main/java/com/laytonsmith/abstraction/bukkit/events/drivers/BukkitBlockListener.java [hunk 1]
```diff
@@ -124,4 +124,10 @@ public void onNotePlay(NotePlayEvent e) {
 		BukkitBlockEvents.BukkitMCNotePlayEvent npe = new BukkitBlockEvents.BukkitMCNotePlayEvent(e);
 		EventUtils.TriggerListener(Driver.NOTE_PLAY, "note_play", npe);
 	}
+
+	@EventHandler(priority = EventPriority.LOWEST)
+	public void onBloackFade(BlockFadeEvent e) {
```
Comment (PseudoKnight) [inline src/main/java/com/laytonsmith/abstraction/bukkit/events/drivers/BukkitBlockListener.java]: Bloack.

**Comment by EntryPointKR** [comment]
"data" means e.getBlock.getData() ?

**Comment by PseudoKnight** [comment]
What block can fade that has data other than 0?

**Comment by EntryPointKR** [comment]
51(fire), others I don't know

**Comment by PseudoKnight** [comment]
Oh, right. I can't think of a reason to prefilter that, though. I think olddata, newtype, and newdata should be removed from prefilters, as they serve no purpose. Only oldtype and world are meaningful prefilters here.

**Comment by EntryPointKR** [comment]
Okay I'll remove it now

**Pull request merged**